Tasks/DeployVisualStudioTestAgent/CreateOfflineTestAgent.ps1

```diff
@@ -53,7 +53,7 @@ function Start-OfflineProcess([string]$language, [string]$sharedFolder) {
     }
     else {
         Write-Host "Downloading the Test execution engine"
-        Download-TestExecutionEngine -Uri "https://testexecution.blob.core.windows.net/testexecution/4182894/TestExecution.zip"
+        Download-TestExecutionEngine -Uri "https://testexecution.blob.core.windows.net/testexecution/3840309/TestExecution.zip"
```

Comment:
file a bug to track to revert the revert

Reply:
not reverting from master